gateway/tests/api/test_v1_serializers.py

```diff
@@ -323,33 +323,3 @@ def test_upload_program_serializer_blocked_dependency(self):
 
         serializer = UploadProgramSerializer(data=data)
         self.assertFalse(serializer.is_valid())
-
```

Comment:
I assume you removed the test because it's not valid anymore. Can we add a new test validating the new logic? 🙏

Reply:
Currently I've only got logic to check that the dependency itself is allowed. It doesn't yet check whether a specific version is allowed or not (that logic is more complicated), so there's no need to test for that right now.

Reply:
Oh, I see, this test is to check the version. I thought it was the test for the previous PR, sorry 👍